### PR TITLE
CalculateAverage_faridtmammadov

### PR DESCRIPTION
**Pull request by faridtmammadov**
#### Check List:
- [x] Tests pass (`./test.sh faridtmammadov` shows no differences between expected and actual outputs)
- [x] All formatting changes by the build are committed
- [x] Your launch script is named `calculate_average_faridtmammadov.sh` (make sure to match casing of your GH user name) and is executable
- [x] Output matches that of `calculate_average_baseline.sh`
* Execution time: 20.925 seconds
* Execution time of reference implementation:  182.758 seconds

### Machine:
* CPU: Apple M2 Pro
* Number of Cores: 10
* RAM: 16GB

### CODE REVIEW

File: src/main/java/dev/morling/onebrc/CalculateAverage_faridtmammadov.java [hunk 1]
```diff
@@ -0,0 +1,188 @@
+package dev.morling.onebrc;
```
Comment (gunnarmorling) [inline src/main/java/dev/morling/onebrc/CalculateAverage_faridtmammadov.java:1]: Please run mvn license:format to add the missing license header.

File: calculate_average_faridtmammadov.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
```
Comment (gunnarmorling) [inline calculate_average_faridtmammadov.sh:1]: Please make this file executable.

Comment (faridtmammadov) [reply calculate_average_faridtmammadov.sh:1]: updated this file with 'git update-index --chmod=+x'

**Comment by gunnarmorling** [comment]
00:17.520. Thx for participating!

**Pull request merged**